From 1659872000da0e923dc612e026cf26f38ddc3e24 Mon Sep 17 00:00:00 2001
From: Luotao Fu <l.fu@pengutronix.de>
Date: Thu, 21 Jan 2010 13:27:13 +0100
Subject: [PATCH] cyclictest: fix wakeup latency tracing entry name

the control entries for wakeup hist latency is put in a directory simply
named as "wakeup", not "wakeup_latency"

Signed-off-by: Luotao Fu <l.fu@pengutronix.de>
---
 src/cyclictest/cyclictest.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/cyclictest/cyclictest.c b/src/cyclictest/cyclictest.c
index eee7e73..0d9704a 100644
--- a/src/cyclictest/cyclictest.c
+++ b/src/cyclictest/cyclictest.c
@@ -463,7 +463,7 @@ static void setup_tracer(void)
 				setkernvar(traceroptions, traceptr[i]);
 		}
 		setkernvar("tracing_max_latency", "0");
-		setkernvar("latency_hist/wakeup_latency/reset", "1");
+		setkernvar("latency_hist/wakeup/reset", "1");
 	} else {
 		setkernvar("trace_all_cpus", "1");
 		setkernvar("trace_freerunning", "1");
-- 
1.6.6

